digitalocean/loadbalancer/resource_loadbalancer_test.go

```diff
@@ -34,6 +34,8 @@ func TestAccDigitalOceanLoadbalancer_Basic(t *testing.T) {
 						"digitalocean_loadbalancer.foobar", "name", name),
 					resource.TestCheckResourceAttr(
 						"digitalocean_loadbalancer.foobar", "region", "nyc3"),
+					resource.TestCheckResourceAttr(
```

Comment:
Curious why the GHA tests didn't fail for this? 😕

Reply:
The full acceptance test suite does not run automatically on PRs, only unit tests. It can take ~4 hours to complete and spins up many resources. We currently run it against main on a schedule and can trigger runs as needed for review.

Reply:
> break out these out into a separate test so that contributor are still able to run these existing acceptance tests

Just to clarify: the default empty `Type` parameter would create a `REGIONAL` load balancer (ie. as we currently support in prod rn), the feature flipper only blocks `Type = "GLOBAL"`: so this test should still pass for other users.